eng/pipelines/libraries/base-job.yml

```diff
@@ -14,23 +15,26 @@ parameters:
   displayName: ''
   testScope: ''
   pool: ''
-  stepName: ''
 
 jobs:
   - template: /eng/common/templates/job/job.yml
     parameters:
-      name: ${{ format('libraries_{0}_{1}_{2}{3}_{4}_{5}', parameters.stepName, parameters.framework, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+      name: ${{ format('libraries_{0}_{1}_{2}{3}_{4}_{5}', parameters.name, parameters.framework, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
       ${{ if eq(parameters.framework, 'netcoreapp') }}:
-        displayName: ${{ format('{0} {1}{2} {3} {4}', parameters.stepName, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
+        displayName: ${{ format('Libraries {0} {1}{2} {3} {4}', parameters.displayName, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.buildConfig) }}
       ${{ if ne(parameters.framework, 'netcoreapp') }}:
-        displayName: ${{ format('{0} {1} {2} {3} {4}', parameters.stepName, parameters.osGroup, parameters.framework, parameters.archType, parameters.buildConfig) }}
+        displayName: ${{ format('Libraries {0} {1} {2} {3} {4}', parameters.displayName, parameters.osGroup, parameters.framework, parameters.archType, parameters.buildConfig) }}
 
       enableTelemetry: ${{ parameters.isOfficialBuild }} # TODO: figure out if it's needed
       container: ${{ parameters.container }}
       helixRepo: dotnet/runtime
       pool: ${{ parameters.pool }}
       variables:
-        - _BuildConfig: ${{ parameters.buildConfig }}
+        - buildScriptFileName: libraries
```

Comment:
Why do some of the variables here get prefixed with `_` and the others do not?

Reply:
We should be prefixing this one as well. I believe he just moved it out from the variables template we have as it was. I usually prefix them with `_` since azure devops propagates the variables as environment variables and that can conflict with MSBuild and do weird stuff you didn't intend MSBuild to do because there was a variable name clash in between yaml variables and MSBuild properties.